### PR TITLE
[ENHANCEMENT] Modified Like Button Functions

### PR DESCRIPTION
**Pull request by yasharth291**
**What kind of change does this PR introduce?**

enhancement

**Summary**

it is related to the issue #235. This PR allows the icon button to change its color from grey to blue on clicking and if clicked again it will again change its color. All the changes told by @Sagar2366  sir is being implemented in this pr and can be assured by the video i am putting below.

https://user-images.githubusercontent.com/55004621/111157926-76993f80-85bd-11eb-82d9-7a0abd39bc5c.mp4

### CODE REVIEW

**Comment by yasharth291** [comment]
@Sagar2366  please could you look into it. By tonight i am going to resolve the issue number #235

**Comment by Sagar2366** [comment]
@yasharth291, Add more functionality to this PR - 
1. Clicking on like button again should remove like for the newsfeed. 
2. Should increase/decrease count of likes if different users like/remove like from the posts
3. Even if there is single like for the newsfeed, show icon in blue.

Also, additionally change the PR title consider above suggestions and description, add snaps/video showing different behaviour before and after adding changes.

**Comment by yasharth291** [comment]
Sure i will add some more functionality in this PR and   make this PR resolve more issues

**Comment by yasharth291** [comment]
Could anyone review it

**Comment by yasharth291** [comment]
The requirement asked for this PR are successfully Completed.

File: lib/views/pages/newsfeed/newsfeed.dart [hunk 1]
```diff
@@ -27,6 +29,9 @@ class _NewsFeedState extends State<NewsFeed> {
   List postList = [];
   String name;
   Timer timer = Timer();
+  bool like = false;
+  String _currentOrgID;
+  Map<String,bool> mp;
```
Comment (jordanliu) [inline lib/views/pages/newsfeed/newsfeed.dart:34]: Unused variable

File: lib/views/pages/newsfeed/newsfeed.dart [hunk 2]
```diff
@@ -27,6 +29,9 @@ class _NewsFeedState extends State<NewsFeed> {
   List postList = [];
   String name;
   Timer timer = Timer();
+  bool like = false;
```
Comment (jordanliu) [inline lib/views/pages/newsfeed/newsfeed.dart:32]: Unused variable

**Review by jordanliu — CHANGES_REQUESTED** [review]

File: lib/views/pages/newsfeed/newsfeed.dart (hunk 1 above)
Comment (yasharth291) [reply lib/views/pages/newsfeed/newsfeed.dart]: resolved

**Comment by yasharth291** [comment]
@jordanliu changes has been done please now can it be merged

**Review by Sagar2366 — APPROVED** [review]

**Comment by yasharth291** [comment]
Closing this and opening new with new changes

**Pull request closed without merging**